docs/docs/kubernetes/cli/cis.md

```diff
@@ -0,0 +1,161 @@
+| 5.4.2  | Consider external secret storage (Manual)                                                              | Consider the use of an external secrets storage and management system, instead of using Kubernetes Secrets directly, if you have more complex secret management needs.                                                                                         |
+| 5.5.1  | Configure Image Provenance using ImagePolicyWebhook admission controller (Manual)                      | Configure Image Provenance for your deployment.                                                                                                                                                                                                                |
+| 5.7.1  | Create administrative boundaries between resources using namespaces (Manual)                           | Use namespaces to isolate your Kubernetes objects.                                                                                                                                                                                                             |
+| 5.7.2  | Ensure that the seccomp profile is set to docker/default in your pod definitions                       | Enable docker/default seccomp profile in your pod definitions.                                                                                                                                                                                                 |
+| 5.7.3  | Apply Security Context to Your Pods and Containers                                                     | Apply Security Context to Your Pods and Containers                                                                                                                                                                                                             |
+| 5.7.4  | The default namespace should not be used                                                               | Kubernetes provides a default namespace, where objects are placed if no namespace is specified for them                                                                                                                                                        |
+
+## CLI Commands
+
+Scan a full cluster and generate a complliance CIS-Benchmark summary report:
+
+```
+$ trivy k8s cluster --compliance=cis --report summary
+```
+
+![k8s Summary Report](../../../imgs/trivy-cis-summary.png)
+
+***Note*** : The `Issues` column represent the total number of failed checks for this control.
+
+
+An additional report is supported to get all of the detail the output contains, use `--report all`
+```
+$ trivy k8s cluster --compliance=cis --report all
+```
+Report also supported in json format examples :
+
+```
+$ trivy k8s cluster --compliance=cis --report summary --format json
+```
+
+```
+$ trivy k8s cluster --compliance=cis --report all --format json
+```
+
+## Custom compliance report
```

Comment:
Seems like the custom report is also described in the NSA page. We may want to add `Overview` under `Kubernetes/CLI/Compliance` and explain that Trivy supports NSA and CIS Benchmark both. Also, we can mention custom reports in that page.